CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### General
 
+- Support fixed bucket histogram aggregation and exporting them with OTLP/Prometheus exporter.
```

Comment:
this should go down in the metrics section, rather than General

Reply:
updated.